md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTemplateAPI.java

```diff
@@ -638,20 +562,19 @@ public void AdjustNextIdsForApply(ProcessTemplate processTemplate) {
             int count_outer = 0;
             for (ProcessTemplate processTemplateForNext : processTemplatesForNext) {
                 LOGGER.debug("processTemplate id= " + processTemplateForNext.getProcessTemplateId());
-                String nextTemplateList[] = processTemplateForNext.getNextProcessTemplateId().split(",");
+                String[] nextTemplateList = processTemplateForNext.getNextProcessTemplateId().split(",");
                 LOGGER.debug("nextTemplateList= " + nextTemplateList[0]);
 
                 processTemplate2 = new ProcessTemplate();
                 LOGGER.debug("processTemplatesForNext.get(count)= " + processTemplateForNext.getProcessId());
-                int count_inner = 0;
-                //        List<Process> processesForNext = s.selectList("call_procedures.select-process-list-for-template-id", processTemplatesForNext.get(count_outer));
+                int countInner = 0;
                 List<Process> processesForNext = processTemplateDAO.selectPListForTemplate(processTemplatesForNext.get(count_outer).getProcessTemplateId());
                 for (Process processForNext : processesForNext) {
 
                     LOGGER.debug("processesForNext= " + processForNext.getProcessId());
                     String nextProcessList = "";
-                    LOGGER.debug("count_inner= " + count_inner);
-                    processTemplate2.setParentProcessId(parentProcessList.get(count_inner));
+                    LOGGER.debug("countInner= " + countInner);
+                    processTemplate2.setParentProcessId(parentProcessList.get(countInner));
                     for (int i = 0; i < nextTemplateList.length; i++) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)